src/components/general/OutcomeCard.tsx

```diff
@@ -198,6 +201,8 @@ const OutcomeCard = (props: OutcomeCardProps) => {
         color={color}
         refProp={cardRef}
       >
+        {refetching && <Loader />}
+
```

Comment:
This works really nicely when changing the scenario but for some reason the loader isn't shown when changing the "Ziel" (also applies to the loaders on the outcome charts). Perhaps that's a separate network request?